### DISCUSSION TITLE
regular expression

### DISCUSSION THREAD

**Discussion by mariannevanharten** [fisharebest/webtrees#4807] [General]
When using Control panel - Manage family trees - Data fixes - Search and replace - regular expression I think something is not working correctly.

By accident I entered in the field **Replacement text** also a backslash before the parentheses
**\(Gld\)**
It results in a string including the backslash.
Now I try to delete the unwanted backslash, so I searched **\\(Gld\\)** and replacement **(Gld)**,
The string is not found in my database...

Is that really a bug or did I make a mistake to correct this?

Best regards,
Marianne

**Comment by fisharebest** [comment]
The backslash has special meaning for the SEARCH TEXT.
But it is just a normal character in the REPLACE TEXT.

> Now I try to delete the unwanted backslash, so I searched \(Gld\) and replacement (Gld),

You should search for `\\\(Gld\\\)`.  `\\` matches a backslash. `\(` matches a bracket.

Or,... use a normal search/replace.  You do not need a regular expression to replace `\(Gld\)` with `(Gld)`.
